serialize.py

```diff
@@ -318,7 +318,8 @@ def main():
     raise Exception('Invalid network input format.')
 
   if args.ft_compression != 'none' and not args.target.endswith('.nnue'):
-    raise Exception('Compression only allowed for .nnue target.')
+    args.ft_compression = 'none'
```

Comment:
oops, i didn't mean to push this change to this PR. forgot there was an open PR with this branch. this was a hacky quick fix to get `--start-from-model` working when using leb128 as the default ft_compression arg. otherwise the trainer tries to serialize to .pt with leb128 compression and crashes upon start.

a cleaner fix may be to explicitly use `--ft_compression=none` when start models are prepared with `--start-from-model`

Reply:
yes, I merged this because I realized my PR had a bug and this one was complete